scripts/translate_locales.py

```diff
@@ -0,0 +1,33 @@
+
+# A script to generate a template which will be used to localize the supported locale name
+# For more inofrmation see https://bugzil.la/859499#c11
+
+from django.conf import settings
+
+from jinja2 import Environment
+
+
+template_string = ("""
+
+This template is automatically generated by scripts/translate_locales.py to make the languages name localizable. Do not edit it manually
+Background: https://bugzil.la/859499#c11
+
+{% for lang in languages %}
+   {{ '{{ _("' + lang + '") }}' }}
+{% endfor %}
+
+""")
+
+
+LANGUAGES = sorted([lang_info.english.encode('ascii', 'ignore')
```

Comment:
I think ``english.encode('ascii', 'ignore')`` is the wrong thing - it will throw away non-ASCII characters.  I'm more comfortable with UTF-8, like ``english.encode('utf8')``